### PR TITLE
Did some mappings in Direction

### PR DESCRIPTION
**Pull request by Kroppeb**
The codec will be added in a later pr.

### CODE REVIEW

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
+	COMMENT
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping]:
```suggestion
```

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 2]
```diff
@@ -28,6 +29,12 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_10143 byId (I)Lnet/minecraft/class_2350;
 		ARG 0 id
 	METHOD method_10144 asRotation ()F
+	METHOD method_10145 toDirectionArray (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping]: this is like listing the closest directions, used by entity facing order i think. Name needs improvement

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 3]
```diff
@@ -64,9 +72,11 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_23224 getRotationQuaternion ()Lnet/minecraft/class_1158;
 	METHOD method_23225 transform (Lnet/minecraft/class_1159;Lnet/minecraft/class_2350;)Lnet/minecraft/class_2350;
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD values ()[Lnet/minecraft/class_2350;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping]:
```suggestion
```
useless entry i think

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 4]
```diff
@@ -64,9 +72,11 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
...
 	CLASS class_2351 Axis
 		FIELD field_11050 BY_NAME Ljava/util/Map;
 		FIELD field_11053 name Ljava/lang/String;
+		FIELD field_23780 ALL [Lnet/minecraft/class_2350$class_2351;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping]:
```suggestion
		FIELD field_23780 VALUES [Lnet/minecraft/class_2350$class_2351;
```

File: mappings/net/minecraft/util/math/Direction.mapping (hunk 3 above)
Comment (Kroppeb) [reply mappings/net/minecraft/util/math/Direction.mapping]: Yeah, I think I did that one by accident

File: mappings/net/minecraft/util/math/Direction.mapping (hunk 4 above)
Comment (Kroppeb) [reply mappings/net/minecraft/util/math/Direction.mapping]: I saw the `Direction.ALL` and tried to be consistent.

File: mappings/net/minecraft/util/math/Direction.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/util/math/Direction.mapping]: Maybe `listClosestDirections`?
```suggestion
	METHOD method_10145 listClosestDirections (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```
the first, second, third names can be retained.

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 5]
```diff
@@ -28,6 +28,12 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_10143 byId (I)Lnet/minecraft/class_2350;
 		ARG 0 id
 	METHOD method_10144 asRotation ()F
+	METHOD method_10145 listClosestDirections (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
+		COMMENT Helper function that returns the 3 given folowed by the
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping:32]:
```suggestion
		COMMENT Helper function that returns the 3 given directions followed by the
```

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 6]
```diff
@@ -28,6 +28,12 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_10143 byId (I)Lnet/minecraft/class_2350;
 		ARG 0 id
 	METHOD method_10144 asRotation ()F
+	METHOD method_10145 listClosestDirections (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping:31]:
```suggestion
	METHOD method_10145 listClosest (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```
Imo `direction` suffix is redundant.

**Pull request closed without merging**